src/commands/push-to-app-collection.yaml

```diff
@@ -55,22 +55,24 @@ steps:
         cd .app-collection && git add helm/<<parameters.app_collection_repo>>-chart/templates/$(cat ../.app_file_name)
   - run: |
         cd .app-collection && git commit -m "$(cat ../.git_commit_message)"
-  - run: |
-        cd .app-collection && \
-        declare -i attempt=0 && \
-        while ! git push; do
-          attempt+=1
-          if ((attempt >= 4)); then
-            printf '%s\n' \
-              "Error pushing app to the collection. See known errors in:" \
-              "https://github.com/giantswarm/architect-orb/blob/master/README.md#push-to-app-collection" \
-              "Giving up after $attempt failures." \
-              >&2
-            exit 1
-          fi
-          sleep 5
-          git pull --rebase --autostash
-        done
+  - run:
+      name: Push to app collection repo
```

Comment:
I just added `name`. This name is not conformant but I didn't want to confuse with prefix when only one step has a name. I also didn't want to pollute diff.